spring-pulsar/src/test/java/org/springframework/pulsar/listener/ConcurrentPulsarMessageListenerContainerTests.java

```diff
@@ -105,6 +96,17 @@ void basicConcurrencyTesting() throws Exception {
 	@Test
 	@SuppressWarnings("unchecked")
 	void exclusiveSubscriptionMustUseSingleThread() throws Exception {
+		MockEnvironment env = setupMockEnvironment(SubscriptionType.Exclusive);
+		ConcurrentPulsarMessageListenerContainer<String> concurrentContainer = env.concurrentContainer();
+
+		concurrentContainer.setConcurrency(3);
+
+		assertThatThrownBy(concurrentContainer::start).isInstanceOf(IllegalStateException.class)
```

Comment:
Nice use/test.

Reply:
The kudos belong to @sobychacko, it's just the git diff that makes it look like new code 😉